rootfs/etc/profile.d/inputrc.sh

```diff
@@ -0,0 +1,7 @@
+# If we can, and do not have one, copy inputrc from localhost.
+[ -f ~/.inputrc ] && [ -f /localhost/.inputrc ] || {
+	cp /localhost/.inputrc ~/.inputrc &&
```

Comment:
Why the copy and not just bind?

Reply:
Any thoughts on symlinking instead?

Reply:
I've implemented your "just bind" suggestion in the next commit.